reframe/frontend/loader.py

```diff
@@ -119,6 +119,10 @@ def recurse(self):
         return self._recurse
 
     def _set_defaults(self, test_registry):
+        if not test_registry:
+            return
```

Comment:
Do we need this?

Reply:
Yes, if we load a file that has only `parameterized_test`s then the registry is empty. We do a similar thing here:

https://github.com/eth-cscs/reframe/blob/da18b68ab7cdcd2057d5b1eed1f77db2b3588631/reframe/frontend/loader.py#L144

Reply:
But when `not test_registry`, this `if` statement has the same effect as the `for` loop below, that's why I don't see the need for it.

Reply:
It is `None`. I could make the test more explicit.